Index: app/src/main/java/com/t3h/myprojectnoteupdate/activity/MenuSettingActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/t3h/myprojectnoteupdate/activity/MenuSettingActivity.java	(date 1566725894738)
+++ app/src/main/java/com/t3h/myprojectnoteupdate/activity/MenuSettingActivity.java	(date 1566725894738)
@@ -47,6 +47,8 @@
     private TextView btndone;
     private ImageView imgclose;
     private Item item;
+    private Date date1;
+
     @Override
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
@@ -59,7 +61,7 @@
         final String title = intent.getStringExtra("title");
         final String content = intent.getStringExtra("content");
         final String date = intent.getStringExtra("date");
-        final Date date1 = AppUtils.ConvertStringToDate(date);
+         date1 = AppUtils.ConvertStringToDate(date);
 //        SimpleDateFormat formatter = new SimpleDateFormat("dd-MM-yyyy");
 //        String formattedDate = formatter.format(date1);
 //        String title = intent.getStringExtra(EditNewActivity.TITLE);
@@ -88,7 +90,7 @@
                 Intent intent = new Intent();
                 intent.putExtra("Title",title);
                 intent.putExtra("Content",content);
-                intent.putExtra("Date",date);
+                intent.putExtra("Date",AppUtils. getFormattedTimeStringMenu(date1));
                 setResult(RESULT_OK,intent);
                 finish();
             }
